CONTRIBUTING.rst

```diff
@@ -49,34 +45,92 @@ remember that SimPEG is maintained on a volunteer basis. If you suggest an
 enhancement, we certainly appreciate if you are also willing to take action
 and start a pull request!
 
+.. _questions:
+
+Questions
+---------
```

Comment:
should this be at the same level as "Issues" (rather than under it)? Since we ask people not to create issues for questions, it might be simpler as its own heading

Reply:
That makes sense.